### PR TITLE
Feature/136 translation all pages

### PR DESCRIPTION
**Pull request by asadbekkholdarov**

### CODE REVIEW

File: src/api/mutations/accommodations/useDeleteAccommodation.ts [hunk 1]
```diff
@@ -21,7 +24,7 @@ export const useDeleteAccommodation = () => {
       await queryClient.invalidateQueries({
         queryKey: [QUERY_KEYS.query.hostedAccommodation, accommodationId],
       });
-      toast.success('Accommodation deleted successfully');
+      toast.success(`${t(ToastMessages.SuccessAccDel)}`);
```
Comment (adadychanka) [inline src/api/mutations/accommodations/useDeleteAccommodation.ts]: It's better not to cut message names

`SuccessAccountDeleted`

File: src/api/mutations/account/useEditAccountMutation.ts [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+  const { t } = useTranslation();
   return useMutation({
     mutationKey: [QUERY_KEYS.mutation.editAccount],
     mutationFn: async (propertyToChange: Partial<Profile>) => {
       await httpClient.patch(ENDPOINTS.account.updateUserProfile(profileId), propertyToChange);
     },
     onSuccess: async () => {
       await queryClient.invalidateQueries({ queryKey: [QUERY_KEYS.query.user] });
-      toast.success(t(ProfileActions.profile_change));
+      toast.success(`${t(ToastMessages.SuccessUpdated)}`);
```
Comment (adadychanka) [inline src/api/mutations/account/useEditAccountMutation.ts]: What was successfully updated? 

It is not understandable looking at the message name. Please, don't shorten message names

File: src/api/mutations/auth/useVerifyEmailMutation.ts [hunk 3]
```diff
@@ -6,17 +6,20 @@ import httpClient from '@src/api/httpClient';
...
+  const { t } = useTranslation();
 
   return useMutation({
     mutationKey: [QUERY_KEYS.mutation.verifyEmail],
     mutationFn: async () => {
       await httpClient.post<string>(ENDPOINTS.auth.verifyEmail, { token });
     },
     onSuccess: () => {
-      toast.success('Email verified');
+      toast.success(`${t(ToastMessages.SuccessEmail)}`);
```
Comment (adadychanka) [inline src/api/mutations/auth/useVerifyEmailMutation.ts]: Same here

File: src/api/httpClient.tsx [hunk 4]
```diff
@@ -63,7 +63,7 @@ async function resErrInterceptor(error: AxiosError<ErrorResponse>) {
         if ('error' in res) {
           store.dispatch(logout());
           httpClient.defaults.headers.common['Authorization'] = '';
-          toast.error('Your session has expired, please sign in again');
+          toast.error(`${i18n.t(ToastMessages.ErrorSessionExpired)}`);
```
Comment (adadychanka) [inline src/api/httpClient.tsx]: The `t` function returns a string. Why do you wrap it in a template string?

Check this issue across all of your changes

File: src/pages/accomodation/Accommodation.tsx [hunk 5]
```diff
@@ -115,17 +117,24 @@ function Accommodation() {
...
+              </Typography>
             </Box>
             <Typography mt={2}>{data?.description}</Typography>
           </Box>
 
           <AmenityList amenities={amenities} />
         </Box>
         <Box flex={0.4}>
-          {ownerId === data.ownerId ? (
+          {ownerId == data.ownerId ? (
```
Comment (adadychanka) [inline src/pages/accomodation/Accommodation.tsx]: Why did you remove strict equality here?

**Review by adadychanka — CHANGES_REQUESTED** [review]

File: src/api/mutations/account/useEditAccountMutation.ts (hunk 2 above)
Comment (adadychanka) [reply src/api/mutations/account/useEditAccountMutation.ts]: Check all changes for this issue, especially utils messages, names should reflect the message

File: src/api/mutations/accommodations/useDeleteAccommodation.ts (hunk 1 above)
Comment (asadbekkholdarov) [reply src/api/mutations/accommodations/useDeleteAccommodation.ts]: sorry, it is orignally SuccessAccommodationDeleted. it was my mistake, and fixed

**Review by adadychanka — APPROVED** [review]

**Review by fasbrothers — APPROVED** [review]
